python/tox.ini

```diff
@@ -26,13 +26,11 @@ code-directories = src
 [coverage]
 coverage = src
 always-on = True
-coverage-config = tox.ini
```

Comment:
I don't recall all the details here -- why remove this?

Reply:
It's a bit fuzzy because I made the change a while ago but I seem to recall it was causing an issue where the debugger was looking for the tox.ini file in the same directory as the `nose2` executable so I would never be able to debug.